target/linux/rockchip/armv8/base-files/etc/board.d/01_leds

```diff
@@ -25,8 +25,8 @@ friendlyarm,nanopi-r5c)
 	;;
 friendlyarm,nanopi-r5s)
 	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth0"
```

Comment:
I'm sorry. I guess you are right, but I am thinking about why this works although we delete the `"green:wan"` label?

Reply:
> I am thinking about why this works although we delete the "green:wan" label?

We have `color` (green) and `function` (wan) properties ;)